Assets/Scripts/Projectile.cs

```diff
@@ -0,0 +1,39 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class Projectile : MonoBehaviour {
+
+    public float damage = 25f;
+    public float speed = 15f;
+
+    private Rigidbody2D self;
+    private GameObject player;
```

Comment:
Why does every projectile need to know about the player? The oncollision block seems to show handle it already.